### DISCUSSION TITLE
打开页面后重新点击菜单如何刷新页面缓存？

### DISCUSSION THREAD

**Discussion by JontyMin** [vbenjs/vue-vben-admin#2289] [Q&A]
我想要在第二次点击左侧菜单时刷新页面缓存，请指导
![004](https://user-images.githubusercontent.com/51729732/196132843-8f46cfe3-047e-4462-b244-d3a0b4885e3f.gif)

**Comment by qiekenol** [comment]
页面不要配置name就不缓存了
